docs/user/features/tags.md

```diff
@@ -11,20 +11,45 @@ You can add tags to your notes to categorize or link notes together.
 There are two ways of creating a tag:
 
 - Adding a `#tag` anywhere in the text of the note, for example: #my-tag1
-- Using the `tags: tag1, tag2` yaml frontmatter [[note property|note-properties]]. Notice `my-tag1` and `my-tag2` tags which are added to this document this way.
+- Using the `tags: tag1, tag2` yaml frontmatter [[note-properties|note property]]. Notice `my-tag1` and `my-tag2` tags which are added to this document this way.
 
-Tags can also be hierarchical, so you can have `#parent/child`.
+Tags can also be hierarchical, so you can have `#parent/child` such as #my-tag3/info.
 
 ## Using *Tag Explorer*
 
-It's possible to navigate tags via the Tag Explorer panel.
-In the future it will be possible to explore tags via the graph as well.
+It's possible to navigate tags via the Tag Explorer panel. Expand the Tag Explorer view in the left side bar which will list all the tags found in current Foam environment. Then, each level of tags can be expanded until the options to search by tag and a list of all files containing a particular tag are shown.
+
+Tags can also be visualized in the Foam Graph Explorer. See [[graph-visualization]] for more info including how to change the color of nodes representing tags.
 
 ## Styling tags
 
-Inline tags can be styled using custom CSS with the selector `.foam-tag`.
+It is possible to customize the way that tags look in the Markdown Preview panel that renders your Foam notes.
+
+1. Create a CSS file within your Foam project, for example in `assets/css/custom-tag-style.css`
```

Comment:
Thanks for adding this customization, super handy.
All Foam related files should by default live in `.foam`, as to not pollute the user's notes.
And I can see how this could be used to further style the rendering in the preview.
So maybe we can have `.foam/assets/css/preview-style.css`?

Reply:
Ah, that's a great idea. I might shorten that path to just `.foam/css/preview-style.css` unless you can think of some other type of asset that might belong under a `.foam/assets` folder?